build/Linux+BSD/portable/Recipe

```diff
@@ -175,7 +175,8 @@ function download_linuxdeploy_component()
 if [[ ! -d "appimagetool" ]]; then
   mkdir appimagetool
   cd appimagetool
-  download_appimage_release AppImage/AppImageKit appimagetool
+  # `12` and not `continuous` because see https://github.com/AppImage/AppImageKit/issues/1060
+  download_appimage_release AppImage/AppImageKit appimagetool 12
```

Comment:
These changes are in 3.5rc and in 3.x, so not an issue, but why then is it listed here?